app/(app)/scan/scan-result.tsx

```diff
@@ -0,0 +1,51 @@
+import { router, useLocalSearchParams } from 'expo-router'
+
+import ContinueButton from '@/components/navigation/ContinueButton'
+import ContentWithAvatar from '@/components/screen-layout/ContentWithAvatar'
+import ScreenViewCentered from '@/components/screen-layout/ScreenViewCentered'
+import { ScanResultEnum } from '@/modules/backend/openapi-generated'
+
+/*
+ * Figma: https://www.figma.com/file/3TppNabuUdnCChkHG9Vft7/paas-mpa?node-id=1300%3A11943&mode=dev
+ */
+
+// TODO may need update depending on changes in figma
+type ScanResultSearchParams = {
+  scanResult: ScanResultEnum
+}
+
+const ScanResultPage = () => {
+  const { scanResult } = useLocalSearchParams<ScanResultSearchParams>()
+
+  return (
+    <ScreenViewCentered
+      options={{ headerTransparent: true }}
+      actionButton={
+        scanResult === ScanResultEnum.NoViolation ? (
+          <ContinueButton onPress={router.back}>
+            {`${scanResult}.actionButtonLabelBack` as string}
+          </ContinueButton>
+        ) : (
+          <ContinueButton variant="negative" onPress={() => router.push('/offence')}>
+            {`${scanResult}.actionButtonLabel` as string}
+          </ContinueButton>
```

Comment:
Is this correct? Looks like translation function keys but without the actual `t` function :D 

Also `as string` doesn't look right but I don't see what is the problem why it is needed.

Reply:
It's still todo... texts are going to be changed and I'm also going to add  translations